tests/gui/conversation/export/test_device.py

```diff
@@ -38,31 +38,6 @@ def test_Device_run_printer_preflight_checks(homedir, mocker, source):
         assert len(print_preflight_check_requested_emissions) == 1
 
 
-def test_TestDevice_run_printer_preflight_checks(homedir, mocker, source):
```

Comment:
Removing the tests corresponding to the not-Qubes OS-specific behavior that's no longer in our production code. (:slightly_smiling_face:)